server/server.js

```diff
@@ -8,27 +8,23 @@ const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
+app.use(cors({ origin: "*" }));
 
-app.set("view engine", "ejs");
+app.set("view engine", "pug");
 app.use(express.json());
 
 app.use("/api/users", userRouter);
-app.use("/api/blogs", blogRouter);
+app.use("/blogs", blogRouter);
 
-// Add this error-handling middleware after all other app.use() and routes calls
 app.use((err, req, res, next) => {
-  console.error(err.stack);
-  res.status(500).send('Something broke!');
+  console.error("Custom Error:", err.stack);
+  res.status(400).send('Something went wrong!');
 });
 
```

Comment:
Avoid extraneous debug functionality in production code. Remove the 'Custom Error:' prefix from the console log to prevent potentially exposing internal details.

```suggestion
  console.error(err.stack);
```